src/PlaybackManager.vala

```diff
@@ -455,4 +456,30 @@ public class Music.PlaybackManager : Object {
         var play_pause_action = (SimpleAction) GLib.Application.get_default ().lookup_action (Application.ACTION_PLAY_PAUSE);
         play_pause_action.set_enabled (current_audio != null);
     }
+
+        public void save_queue () {
```

Comment:
Doesn't look like this is being called in this branch.  In #808 this appears to be part of `on_items_changed ()` which makes sense to me